### DISCUSSION TITLE
What's needed for a new release?

### DISCUSSION THREAD

**Discussion by wlach** [pallets-eco/flask-mail#215] [Announcements]
I think it would be good to do up a new release partly just to refresh pypi and point people to a new location. So far we haven't really changed that much other than updating the build system, fixing some very minor bugs (that people probably didn't even run into) and revamping the docs. We now require Python 3.8+ but that seems very reasonable in 2024.

My weak proposal would be to do the following:

- [x] Merge some version of https://github.com/pallets-eco/flask-mail/pull/89 since it seems like a genuine (pretty obvious) bug which is easy to fix (the original author hasn't responded to me so far so we might need to recreate it, not that big a deal)
- [x] Fix versioning so we're not hardcoding it in 2 or 3 different places (maybe crib the approach used in flask: https://github.com/pallets/flask/blob/255c8d66af6daff3eaa063ea0819e185d4d1e214/src/flask/__init__.py#L58)
- [x] Update CHANGES

Then just release a 0.10.0 with these releases (I figured we could save a major version bump for a significant change in functionality / backwards compatibility -- I believe @davidism had some ideas on how this library could be simplified/revamped)

Thoughts?

**Comment by davidism** [comment]
I think a 0.10 release with our refresh as well as a few clear bug fixes would be good. I'm not sure what 1.0 would look like quite yet (although if other pallets-eco members have an idea that's fine instead).

I think we should add #110. It's technically a breaking change, but there's no way to deprecate the current behavior, we're zero ver, and the current behavior doesn't have a meaningful use.

**Comment by davidism** [comment]
I've created a 0.10.0 milestone to collect all the issues and PRs, and added everything done over the last few days. https://github.com/pallets-eco/flask-mail/milestone/1

I've also created a 1.0.0 milestone for organizing that release. https://github.com/pallets-eco/flask-mail/milestone/2

**Comment by davidism** [comment]
I'm not clear that #88 and #89, and #93 and #101 are correct. I've closed them for now.

**Comment by wlach** [comment]
@davidism released 0.10.0, so we can call this done: https://github.com/pallets-eco/flask-mail/releases/tag/0.10.0
